spring-5-webflux/src/test/java/com/baeldung/spring/serverconfig/TimeoutLiveTest.java

```diff
@@ -44,28 +49,22 @@ public void setup() throws SSLException {
 
     @Test
     public void shouldTimeout() {
-        try {
-            webTestClient.get()
-              .uri("/timeout/{timeout}", 3)
-              .exchange();
-            Assert.fail();
-        } catch (ReadTimeoutException ignore) {
-        }
+        exception.expect(ReadTimeoutException.class);
+        webTestClient.get()
+          .uri("/timeout/{timeout}", 3)
+          .exchange();
+        Assert.fail();
```

Comment:
this can now be removed because if the exception is not thrown, JUnit it will fail it since it was expecting it

Reply:
Good point.  I forgot to remove that.  It is done.